packages/api/src/logger/index.ts

```diff
@@ -159,10 +159,10 @@ export const logLevel: LevelWithSilent | string = (() => {
  *
  * @see {@link https://github.com/pinojs/pino/blob/master/docs/api.md}
  */
-export const defaultLoggerOptions: LoggerOptions = {
+export const defaultLoggerOptions = {
   level: logLevel,
   redact: redactionsList,
-}
+} satisfies LoggerOptions
 
```

Comment:
new TS 4.9 [feature](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator), lets you say "this object must conform to this type" but doesn't force the type to be that type to other objects.

What this means is that I can access `defaultLoggerOptions.level` without a `?` because it's not `LoggerOptions`'s `string | undefined` but this object's `string`.